scripts/ps_dm_component.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+#check that data masking plugin is disabled
+DM_PLUGIN=$(mysql -NBe "SELECT PLUGIN_NAME, PLUGIN_STATUS FROM INFORMATION_SCHEMA.PLUGINS WHERE PLUGIN_NAME = 'data_masking';" | grep -c ACTIVE)
```

Comment:
Maybe just SELECT PLUGIN_STATUS. I dont see any significant use of fetching PLUGIN_NAME